aclk/aclk_common.c

```diff
@@ -0,0 +1,42 @@
+#include "aclk_common.h"
+
+struct {
+    ACLK_PROXY_TYPE type;
+    const char *url_str;
+} supported_proxy_types[] = {
+    { .type = HTTP,     "http://"    },
```

Comment:
Should we not add `https://` to the list?

Reply:
Hi that will be another PR, i removed http from this PR completely

Reply:
All right, thank you!